Userland/Applications/Calculator/KeypadValue.h

```diff
@@ -43,6 +54,6 @@ class KeypadValue {
     // m_decimal_places would be 2, because when you shift -12355 2 digits to the right, you get -123.55.
     // This way, most operations don't have to be performed on doubles, but can be performed without loss of
     // precision on this class.
-    i64 m_value { 0 };
-    u8 m_decimal_places { 0 };
+    Checked<i64> m_value { 0 };
+    Checked<u8> m_decimal_places { 0 };
```

Comment:
This is very, very weird. Why are the members themselves `Checked`? Shouldn't we only have to deal with overflow checking while we're performing the calculations? When we simply have "some value that the Keypad can take", we should not admit the possibility of some internal variables being in an invalid state.